sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -55,6 +64,9 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
         });
 
         this.itemViews = itemCards;
+        if (this.itemViews.length > 0) {
```

Comment:
Yes, because now that there's no item_counts, you can't live update the notion of "New" for a category when clicking on items in a feed or for a feed when clicking on items in a category. This makes it such that when you open a new itemsView, it checks if all the items are read and updates it accordingly.